code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -50,11 +50,9 @@
 	if(disguise == null)
 		return
 	stealth_active = 1
-	if(istype(src.loc, /mob/living/carbon/human))
-		var/mob/living/carbon/human/M = src.loc
-		addtimer(GLOBAL_PROC, "anim", 0, FALSE, M.loc, M, 'icons/mob/mob.dmi',
-			null, "cloak", null, M.dir)
-
+	if(istype(loc, /mob/living/carbon/human))
+		var/mob/living/carbon/human/M = loc
+		new /obj/effect/overlay/temp/dir_setting/abductor_cloak(get_turf(M), M.dir)
```

Comment:
oh, and this should be `PoolOrNew(/obj/effect/overlay/temp/dir_setting/ninja/cloak, list(get_turf(M), M.dir))`